CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [v5.0.0] - 2019-11-12
+### Changed
+- Default coding style is now PSR-12
+- Default code style fixer is now `squizlabs/php_codesniffer`
+### Added 
+- Support for phpspec v5 and v6
+### Removed
+- Support for phpspec v4
```

Comment:
This is still a TODO and depends on https://github.com/karriereat/phpspec-matchers/pull/14.